src/datepicker/datepicker-config.ts

```diff
@@ -10,12 +10,14 @@ import {NgbDateStruct} from './ngb-date-struct';
 @Injectable()
 export class NgbDatepickerConfig {
   dayTemplate: TemplateRef<DayTemplateContext>;
+  displayMonths = 1;
   firstDayOfWeek = 1;
   markDisabled: (date: NgbDateStruct, current: {year: number, month: number}) => boolean;
   minDate: NgbDateStruct;
   maxDate: NgbDateStruct;
   outsideDays: 'visible' | 'collapsed' | 'hidden' = 'visible';
   showNavigation = true;
+  showNavigationSelect = true;
```

Comment:
hmm...

`showNavigation` → hides / shows navigation completely, leaving only months to be displayed
`showNavigationSelect` → toggles month and year select boxes

Actually you're right, now I think of merging them into:

```ts
navigation: 'select' | 'arrows' | 'hidden';
```

P.S. not sure about naming... ideas?